.travis.yml

```diff
@@ -7,7 +7,7 @@ compiler:
 
 before_script:
   - sudo apt-get update -y
-  - sudo apt-get install qemu-kvm libxen-dev libseccomp-dev -y
+  - sudo apt-get install qemu-kvm libxen-dev libseccomp-dev bats genisoimage -y
```

Comment:
Let's install sudo here just in case since we use it later on.

Reply:
added